### PR TITLE
refactor: use new swapper for keep3r job

### PR DESCRIPTION
**Pull request by 0xsambugs**
We are now using the new swapper for the Keep3r job

In doing so, we reduced the work job by 19k! And we were only swapping one pair 💪

### CODE REVIEW

File: test/integration/DCAKeep3rJob/keep3r-job.spec.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ contract('DCAKeep3rJob', () => {
   let WETH: IERC20, K3PR: IERC20;
 
   let DCAKeep3rJob: DCAKeep3rJob;
-  let DCAHubSwapper: DCAHubSwapper;
+  let DCAHubSwapper: ThirdPartyDCAHubSwapper;
```
Comment (0xged) [inline test/integration/DCAKeep3rJob/keep3r-job.spec.ts:28]: Any chance of changing the name to the 3PartySwapper so the test is more explicity?

```suggestion
  let DCAHubSwapper: ThirdPartyDCAHubSwapper;
```

File: test/integration/DCAKeep3rJob/keep3r-job.spec.ts [hunk 2]
```diff
@@ -151,29 +150,27 @@ contract('DCAKeep3rJob', () => {
...
-    });
+    const swapTx = await DCAHub.populateTransaction.swap(
+      [USDC_ADDRESS, WETH_ADDRESS],
+      [{ indexTokenA: 0, indexTokenB: 1 }],
+      DCAHubSwapper.address,
+      DCAHubSwapper.address,
+      [0, 0],
+      bytes,
+      []
+    );
```
Comment (0xged) [inline test/integration/DCAKeep3rJob/keep3r-job.spec.ts:169]: 🔥

**Review by 0xged — APPROVED** [review]
Great work Sam!

File: test/integration/DCAKeep3rJob/keep3r-job.spec.ts (hunk 1 above)
Comment (0xsambugs) [reply test/integration/DCAKeep3rJob/keep3r-job.spec.ts:28]: What do you want to rename exactly?

Comment (0xged) [reply test/integration/DCAKeep3rJob/keep3r-job.spec.ts:28]: Oops.

```suggestion
  let thirdPartySwapper: ThirdPartyDCAHubSwapper;
```

Comment (0xsambugs) [reply test/integration/DCAKeep3rJob/keep3r-job.spec.ts]: Done!

**Pull request merged**